controller/scheduler/fleet.py

```diff
@@ -158,6 +168,10 @@ def _create_container(self, name, image, command, unit, **kwargs):
             f['value'] = f['value'].format(**l)
         # prepare tags only if one was provided
         tags = kwargs.get('tags', {})
+        # this way, if we specify the az tag it will be higher priority then the
+        # cross az feature.
+        azs.update(tags)
```

Comment:
actually, we can't, the update function in python updates the dict in place, it does not return the dict. 
for the result of your command would be tags = None.